src/content/SpeedDial/speeddial.playground.jsx

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import { SpeedDial } from "react-native-elements";
+import Playground from "../../components/playground";
+import { useView, PropTypes } from "react-view";
+
+const SpeedDialPlayground = () => {
+  const params = useView({
+    componentName: "SpeedDial",
+    props: {
+      children: {
+        value: `
+        <SpeedDial.Action
+        icon={{ name: 'add', color: '#fff' }}
+        title="Add"
+        onPress={() => console.log('Add Something')}
+        />
+        <SpeedDial.Action
+            icon={{ name: 'delete', color: '#fff' }}
+            title="Delete"
+            onPress={() => console.log('Delete Something')}
+            />
+            `,
+      },
+      isOpen: {
+        type: PropTypes.Boolean,
+        value: false,
+      },
+      openIcon: {
+        type: PropTypes.Object,
+        value: `{ name: 'close', color: '#fff' }`,
+      },
+      onOpen: {
+        type: PropTypes.Function,
+        value: `() => console.log("onOpen()")`,
+      },
+      onClose: {
+        type: PropTypes.Function,
+        value: `() => console.log("onClose()")`,
+      },
+      transitionDuration: {
+        type: PropTypes.Number,
+        value: 150,
+      },
+      icon: {
+        type: PropTypes.Object,
+        value: `{ name: 'edit', color: '#fff' }`,
+      },
+    },
+
+    scope: {
+      SpeedDial,
+    },
+    imports: {
+      "react-native-elements": {
+        named: ["SpeedDial"],
+      },
+    },
+  });
+
+  return (
+    <React.Fragment>
+      <Playground params={params} />
+    </React.Fragment>
+  );
+};
+
+export default SpeedDialPlayground;
```

Comment:
Can you increase height, so that actions become visible

Reply:
Sure. will do today